osmnx/buildings.py

```diff
@@ -165,7 +165,7 @@ def create_buildings_gdf(polygon=None, north=None, south=None, east=None,
                 nodes = result['nodes']
                 try:
                     polygon = Polygon([(vertices[node]['lon'], vertices[node]['lat']) for node in nodes])
-                except:
+                except Exception as e:
```

Comment:
Hope not too much backseat driving but I think the two possible errors here would be `ValueError` and `AssertionError`, right? Any others. Just a thought if a more specific error catching was desired - explicitly handle those errors related to polygon parsing and raise any others (example would be if there was a type in line 167 that would be silent).